docs/source/sycamore/transforms/materialize.md

```diff
@@ -0,0 +1,62 @@
+The `materialize` transform writes out documents up to that point, marks the materialized path
+as successful if execution is successful, and allows for reading from the materialized data as
+a source.
+
```

Comment:
This is helpful if you're using a notebook and using show() and take() functions to inspect your data in the middle of a Sycamore script. Using `materialize` prevents Sycamore from recomputing after these functions.

Reply:
done.